src/frontend/src/pages/GanttPage/GanttChart.tsx

```diff
@@ -50,6 +50,12 @@ const GanttChart = ({
 
         if (!tasks) return <></>;
 
+        // Sorting the work packages of each project based on their start date
```

Comment:
do you need this for your functionality? I see what you're getting at, but I think keeping them in wbs order would be better for now

Reply:
The reason I did this is because sometimes the work packages would not be displayed in the right order. Like if you see this one, the second work package is below the third one (in yellow the video i took of it before isn't too good)
<img width="1341" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/76594216/4f978285-07bd-48af-b97d-7893eb635523">

Reply:
Ohhh I see, if it's possible to sort it by wbsnum let's do that, otherwise this code is fine

Reply:
https://github.com/Northeastern-Electric-Racing/FinishLine/assets/76594216/883b25ef-e0f2-4fa6-b0b1-91488413e658

Just sorted by WBS nums here. 1.1.2 starts earlier but has a later work package number so it is below the wp with wbs 1.1.1 Is this expected or should I revert it back to sorting by startdate

Reply:
Ahhh no you're right we should prioritize date, that looks wack as hell, thank u for trying it out